src/components/CorporateCampaign/LoanSearch/LoanSearchFilters.vue

```diff
@@ -19,196 +19,220 @@
 			</div>
 
 			<div v-if="showLoanDisplayToggle" class="loan-filters__loan-display">
-				<kv-pill-toggle
-					id="pill"
-					:options="[
-						{
-							title: 'Rows',
-							key: 'rows',
-						},
-						{
-							title: 'Grid',
-							key: 'grid',
-						},
-					]"
+				<div
+					class="tw-flex tw-cursor-pointer tw-items-center"
+					id="view-toggle"
 					:selected="activeLoanDisplay"
-					@pill-toggled="(val) => { $emit('set-loan-display', val === 'rows') }"
-				/>
+					@click-toggle="(val) => { $emit('set-loan-display', val === 'rows') }"
+				>
+					<h4 class="tw-text-h4 tw-font-medium tw-text-action">
+						Row View
+					</h4>
+					<kv-icon
+						class="tw-h-5 tw-mr-1 tw-inline-block"
+						name="list-green"
+						src="@/assets/icons/inline/list-green.svg"
+						:icon="mdiLandRowsHorizontal"
+					/>
+					<span class="divider"></span>
+					<h4 class="tw-text-h4 tw-font-medium tw-text-action">
+						Grid View
+					</h4>
+					<kv-icon
+						class="tw-h-5 tw-mr-1 tw-inline-block"
+						name="grid-green"
+						src="@/assets/icons/inline/grid-green.svg"
```

Comment:
Looks like the `src` can be removed from the `kv-icon` usages in this file, and the associated `svg` files removed from this PR.